src/main/java/org/jabref/logic/pdf/PdfAnnotationImporter.java

```diff
@@ -43,24 +38,22 @@
      * @return a list with the all the annotations found in the file of the path
      */
     @Override
-    public List<FileAnnotation> importAnnotations(final Path path, final BibDatabaseContext context) {
+    public List<FileAnnotation> importAnnotations(final Path path) {
 
-        Optional<Path> validatePath = validatePath(path, context);
-        if (!validatePath.isPresent()) {
+        if (!validatePath(path)) {
             // Path could not be validated, return default result
             return Collections.emptyList();
         }
 
-        Path validPath = validatePath.get();
-
         List<FileAnnotation> annotationsList = new LinkedList<>();
-        try (PDDocument document = PDDocument.load(validPath.toString())) {
+        try (PDDocument document = PDDocument.load(path.toString())) {
             List pdfPages = document.getDocumentCatalog().getAllPages();
             for (int pageIndex = 0; pageIndex < pdfPages.size(); pageIndex++) {
                 PDPage page = (PDPage) pdfPages.get(pageIndex);
                 for (PDAnnotation annotation : page.getAnnotations()) {
-                    if (annotation.getSubtype().equals(FDFAnnotationHighlight.SUBTYPE)) {
-                        annotationsList.add(createHighlightAnnotations(pageIndex, page, annotation));
+                    if (annotation.getSubtype().equals(FDFAnnotationHighlight.SUBTYPE) ||
```

Comment:
You should change the order of equals, the constant first and then the annotation.getsubType as it theoretically could ne null and would result in an NPE.